tests/test_search.py

```diff
@@ -338,10 +338,19 @@ def populate_search_index(self, index_document: dict, count: int) -> list:
             bundle_id = f"{bundle_uuid}.{version}"
             bundle_url = (f"https://127.0.0.1:{self.app._port}"
                           f"/v1/bundles/{bundle_uuid}?version={version}&replica={self.replica_name}")
-            es_client.index(index=self.dss_index_name,
-                            doc_type=ESDocType.doc.name,
-                            id=bundle_id,
-                            body=index_document)
+            if i == count - 1:
```

Comment:
Why this change?

Reply:
I was running into an issue where the test where complete faster than the shards where updating during the `clear_test_indexes` operation even with refresh=true. This was causing the incorrect number of documents to populate the index. To remedy I make sure the index is refreshed after the last documents is index.

Reply:
I don't know if this might be better.  Will leave to your discretion:

```
es_client.index(
  index=self.dss_index_name,
  doc_type=ESDocType.doc.name,
  id=bundle_id,
  body=index_document,
  refresh=(i == count - 1)
)
```

I am pretty weirded out by the fact that you're seeing the behavior you're describing, but I lack the ES expertise to make any judgement on it.

Certainly, you will want to leave a comment explaining this, because otherwise people will be very confused.

Reply:
I like it. Also I'll add in a comment.